src/lesson1/Lesson1Ex6.java

```diff
@@ -0,0 +1,16 @@
+package lesson1;
+
+import java.util.Scanner;
+
+public class Lesson1Ex6 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int x = 8;
+        int y;
+        System.out.println("Enter the number 8");
+        x = sc.nextInt();
+        for(y = 1; y <= 10; y++){
```

Comment:
for (int y = 1; ....)

y-n imast chuni for-iic durs haytarares

u himnakanum for-i mej popoxakani anuny i, j, k,.. sranq en dnum, yndunvac praktika a, y anuny poxi i-ov